src/app/screens/Auth/screens/Login/components/Header/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Image } from 'react-native';
+import HEADER from '@assets/illustrations/headerLogin.png';
```

Comment:
If it is an illustration, the name has to start with 'il' 
for example ilHeaderLogin.png

Reply:
changed with correct name